jme3-core/src/main/java/com/jme3/shadow/SpotLightShadowRenderer.java

```diff
@@ -60,38 +60,37 @@
  * mapping.<br> for more information on this read <a
  * href="http://http.developer.nvidia.com/GPUGems3/gpugems3_ch10.html">http://http.developer.nvidia.com/GPUGems3/gpugems3_ch10.html</a><br>
  *
- * @author Rémy Bouquet aka Nehon
+ * @author Nehon
  */
 public class SpotLightShadowRenderer extends AbstractShadowRenderer {
 
-    protected Camera shadowCam;    
+    protected Camera shadowCam;
     protected SpotLight light;
+    protected Camera[] cameras = new Camera[1];
     protected Vector3f[] points = new Vector3f[8];
-    //Holding the info for fading shadows in the far distance 
-   
+    protected Vector3f tempVec = new Vector3f();
 
     /**
-     * Used for serialization use SpotLightShadowRenderer#SpotLightShadowRenderer(AssetManager assetManager, int shadowMapSize)
+     * For serialization only. Do not use.
      */
     protected SpotLightShadowRenderer() {
         super();
     }
     
     /**
-     * Create a SpotLightShadowRenderer This use standard shadow mapping
+     * Creates a SpotLightShadowRenderer.
      *
-     * @param assetManager the application asset manager
-     * @param shadowMapSize the size of the rendered shadowmaps (512,1024,2048,
-     * etc...) The more quality, the fewer fps.
```

Comment:
I would add back the comment mentioning "The more quality, the fewer fps." however that could certainly be worded better.

Maybe somthing like "Larger shadowmaps not only use more memory but also will result in lower the fps"

Most users would expect a larger image to use more memory (so that may not be important to add) but it is still important to mention that the higher texture correlates with lower framerate, (since that is not always the case with textures and is useful to know here)